### ISSUE TITLE
Feature request: Option to listen to mouse click events

### ISSUE THREAD

**Issue by gandalf3** [alols/xcape#51]
In my [use case](http://unix.stackexchange.com/a/215259/86821) I'm using <kbd>Super</kbd> on it's own to start dmenu, and <kbd>Super</kbd> in combination with mouse 1 and mouse 3 (left and right click) for window moving/resizing. 

The problem occurs when I move windows around fast enough that I release <kbd>Super</kbd> before the timeout is up (quite reasonable when doing small tweaks to window size etc.). When this happens, xcape ends up calling the openbox shortcut and running dmenu *even though* I pressed some mouse buttons while <kbd>Super</kbd> was held.

I tried fiddling with the timeout a bit, and it *kind of* works, but either it still occasionally starts dmenu at undesired times, or I have to stab <kbd>Super</kbd> in a really staccato fashion when I actually want dmenu.

Would it be possible to have xcape listen for mouse button press events in addition to the way it currently listens for keypress events? 

**TL;DR:** Would it be possible to make it so that when <kbd>Super</kbd>+<kbd>Left Mouse</kbd> is pressed within the timeout, the alternate keypresses are *not* generated?

**Comment by alols** [comment]
This was a bug, not a missing feature :) thanks for pointing it out. Please reopen if the new version doesn’t work for you.

**Comment by gandalf3** [comment]
Wow, that was fast! Works perfectly, thanks :)
